user-registration-app/src/app/views/registration-form/registration-form.component.ts

```diff
@@ -240,37 +180,16 @@ export class RegistrationFormComponent implements OnInit {
       )
       .subscribe(() => {
         this.router.navigate([AppConstants.PIN_ENDPOINT])
-          .then(() => this.progress.complete());
+          .then(() => this.progress.ref().complete());
       });
   }
 
-  private showError(message: string, level: number) {
-    switch (level) {
-      case 0:
-        this.alertsService.success(message);
-        break;
-      case 1:
-        this.alertsService.info(message);
-        break;
-      case 2:
-        this.alertsService.warning(message);
-        break;
-      case 3:
-        this.alertsService.danger(message);
-        break;
-      default:
-        this.alertsService.warning(message);
-    }
-  }
-
-  fileAdded(event) {
+  fileAdded(event: any) {
     this.registrationForm.resume = event.target.files[0];
   }
 
   error() {
-    $('html, body').animate({
-      scrollTop: 0,
-    },                      1000);
+    window.scrollTo({ top: 0, behavior: 'smooth' });
```

Comment:
weird question and I have no idea if this actually happens already or not but is there anyway we could make this more specific and if we know what section they didn't fill out (phone number, uni, etc.) we can scroll them to that section instead of just to the top? @rsquared226 this might be something cool to try and implement (assuming it does not already happen lmao)

Reply:
Oooh this is a good idea, I assumed it already did that but we can do that

Reply:
Was this resolved because it got implemented or some other reason?